routes/auths.js

```diff
@@ -14,4 +15,12 @@ router.post('/', validateLoginCredentials, async (req, res, next) => {
   }
 });
 
+router.post('/add', async (req, res) => {
+  const user = req.body;
+  const addeduser = await shoutout.create(user);
```

Comment:
You may consider sticking to the adopted variable naming convention for the project, by changing `addeduser` to `addedUser`

Reply:
I have changed this accordingly